packages/flutter/lib/src/painting/decoration_image.dart

```diff
@@ -268,7 +269,7 @@ class DecorationImagePainter {
       canvas.restore();
   }
 
-  void _imageListener(ImageInfo value, bool synchronousCall) {
+  void _onImage(ImageInfo value, bool synchronousCall) {
```

Comment:
This should be `handleImage` or some such, see style guide

Reply:
Done